applications/DelaunayMeshingApplication/python_scripts/domain_utilities.py

```diff
@@ -160,9 +160,9 @@ def AddBoundaryNodesToFluidDomains(self,model_part):
             print("::[--Domain Utilities-]:: Add boundary nodes to fluid domains ")
```

Comment:
Just wondering if the changes in this file might affect other users of the `DelaunayMeshingApplication`.

Reply:
@rubenzorrilla thanks for the observation!
I think that nobody but PfemFluid people can use that function, anyway I've now moved the transfer process from the PfemFluid to Delaunay Application to resolve possible conflicts.